shell/components/LocaleSelector.vue

```diff
@@ -65,70 +71,35 @@ export default {
 <template>
   <div>
     <div v-if="mode === 'login'">
-      <div
-        v-if="showLocale"
-        role="menu"
-        :aria-label="t('locale.menu')"
-        class="locale-login-container"
-        tabindex="0"
-        @click="openLocaleSelector"
-        @blur.capture="closeLocaleSelector"
-        @keyup.enter="openLocaleSelector"
-        @keyup.space="openLocaleSelector"
-      >
-        <v-dropdown
-          popperClass="localeSelector"
-          :shown="isLocaleSelectorOpen"
-          placement="top"
-          distance="8"
-          skidding="12"
-          :triggers="[]"
-          :autoHide="false"
-          :flip="false"
-          :container="false"
-          @focus.capture="openLocaleSelector"
+      <rc-dropdown v-if="showLocale">
```

Comment:
Shouldn't this be using the new action drop down?

Reply:
My reasons for taking this approach are in the description. I chose the component-driven approach because the existing implementation of the locale selector more closely matches this pattern and requires less refactoring.

Reply:
We could potentially replace the implementation in this PR with the new action menu after #13505 merges.